modules/swagger-codegen/src/main/resources/_common/LICENSE.mustache

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
If a template file doesn't need to be processed through mustache, you can change the `.mustache` extension to something else to skip running it through the processor. [See DefaultCodegen](https://github.com/mateuszmackowiak/swagger-codegen/blob/d82c77da34a29cd9bba71dfde57c9479ae39ad7f/modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultGenerator.java#L550).

This obviously won't be a significant performance gain, but it's useful to know in case we find some text that breaks the mustache template processor.